web/src/inbox_ui.ts

```diff
@@ -1195,9 +1345,18 @@ function is_list_focused(): boolean {
 }
 
 function get_all_rows(): JQuery {
```

Comment:
Maybe add a docstring to this function saying what at a high level it's supposed to be.